packages/core/src/core/deploy/deploy_summary.ts

```diff
@@ -71,6 +76,12 @@ const summarizeAdditionChange = (
     const deployResult = diffBetweenRequestedAndApplied.every(isAdditionChange)
       ? 'success'
       : 'partial-success'
+    if (deployResult === 'partial-success') {
+      log.debug('Addition Summary for change id %s returned partial-success result', {
+        requestedChangeName,
+        diffs: toTrimmedChangeValue(diffBetweenRequestedAndApplied),
+      })
```

Comment:
this is not the style of logs we use in the OSS, extra params are not logged in text format (and here specifically also it won't replace the `%s` in the string which I think you wanted to replace)

this should probably be
```suggestion
      log.debug('Addition Summary for change id %s returned partial-success result, diff between requested and applied: %o',
        requestedChangeName,
        diffBetweenRequestedAndApplied,
      )
```
I think we don't have to explicitly call `inspect` here, this is the syntax of `%o` anyway IIRC, might be good to verify that though...

Reply:
I saw other places do that already, for example https://github.com/salto-io/salto/blob/main/packages/adapter-utils/src/utils.ts#L1268

Any objections to keeping the same style? `%s, inspectValue(x)`

Reply:
Since I know printing `Values` to log is sensitive, I prefer to call `inspectValue` explicitly.